xdsl/dialects/x86/assembly.py

```diff
@@ -106,3 +106,10 @@ def parse_type_pair(parser: Parser) -> SSAValue:
     parser.parse_punctuation(":")
     type = parser.parse_type()
     return parser.resolve_operand(unresolved, type)
+
+
+def source_with_mask_str(reg_in: SSAValue, mask: SSAValue, z: UnitAttr | None) -> str:
```

Comment:
nit:
```suggestion
def masked_source_str(reg_in: SSAValue, mask: SSAValue, z: UnitAttr | None) -> str:
```
Can you please add a doc comment and a unit test?